test/integration/scheduler/filters/filters_test.go

```diff
@@ -41,6 +43,8 @@ var (
 	createNamespacesWithLabels   = testutils.CreateNamespacesWithLabels
 	createNode                   = testutils.CreateNode
 	createPausePod               = testutils.CreatePausePod
+	createPVC                    = testutils.CreatePVC
+	createPV                     = testutils.CreatePV
```

Comment:
let's not add more aliases. This was just a temporary measure to facilitate splitting integration tests in multiple packages.

we should probably get rid of the existing aliases if we have some time.